test/suite-config/repo-client/docker-compose.override.yml

```diff
@@ -33,6 +33,7 @@ services:
     restart: always
     volumes:
       - ./suite-config/repo-client/client/LocalSettings.php.template:/LocalSettings.php.template
+      - ./suite-config/repo-client/LocalSettings.php:/var/www/html/LocalSettings.d/WikibaseLocalMedia.php # disable WikibaseLocalMedia
```

Comment:
So this is mounted only to serve as a dummy to override WikibaseLocalMedia.php? That seems to point to another problem, see above.

Reply:
Yeah this is hax but as the other comment says did try it with out any success. 

Figured client only out-of-the-box setups would a be rare sight and if the user have reached that point disabling an extension would probably be something they are able to do. But lets fix this.